packages/components/src/components/popover/popover.tsx

```diff
@@ -29,38 +30,56 @@ const Popover = ({
     relative_render = false,
     should_disable_pointer_events = false,
     should_show_cursor,
-    window_border,
     zIndex = '1',
     data_testid,
     arrow_styles,
 }: React.PropsWithChildren<TPopoverProps>) => {
     const ref = React.useRef<HTMLDivElement | undefined>();
     const [popover_ref, setPopoverRef] = React.useState<HTMLDivElement | undefined>(undefined);
-
+    const [is_bubble_visible, setIsBubbleVisible] = React.useState(false);
+    const {
+        ui: { is_mobile },
+    } = useStore();
```

Comment:
currently, components in `components` package are using a deprecated `isMobile()` but we are aiming to switch to `is_mobile` from ui-store. Is it ok to import 'useStore' from '@deriv/stores' here? I can see '@deriv/stores''s listed among dependencies of the `components` package, that's why I used it. Open to suggestions :)